src/hotspot/cpu/x86/macroAssembler_x86_arrayCopy_avx3.cpp

```diff
@@ -214,8 +214,8 @@ void MacroAssembler::copy32_masked_avx(Register dst, Register src, XMMRegister x
   mov64(temp, -1L);
   bzhiq(temp, temp, length);
   kmovql(mask, temp);
-  evmovdqu(type[shift], mask, xmm, Address(src, index, scale, offset), Assembler::AVX_256bit);
-  evmovdqu(type[shift], mask, Address(dst, index, scale, offset), xmm, Assembler::AVX_256bit);
+  evmovdqu(type[shift], mask, xmm, Address(src, index, scale, offset), true, Assembler::AVX_256bit);
```

Comment:
Should the merge parameter be set to false for load here?

Reply:
DONE.